src/app/view-modes/text-versions/text-versions.component.ts

```diff
@@ -19,15 +19,16 @@ export class TextVersionsComponent implements OnInit {
   public versionsPanelItem: GridsterItem = { cols: 1, rows: 1, y: 0, x: 1 };
   public versionsOptions: GridsterConfig = {};
 
-  private defaultEditionLevel = AppConfig.evtSettings.edition.availableEditionLevels?.filter((e => !e.disabled))[0];
-  public currentEditionLevel = this.route.params.pipe(
-    map((params) => params.edLvl ?? this.defaultEditionLevel?.id),
-    distinctUntilChanged((x, y) => x === y),
+  public currentPage$ = this.evtStatusService.currentStatus$.pipe(
+    map(({ page }) => page.id),
+  );
+
+  public currentEditionLevel$ = this.evtStatusService.currentStatus$.pipe(
+    map(({ editionLevels }) => editionLevels[0]),
```

Comment:
Can `editionLevels` have zero length?

Reply:
Yes, but if zero length, first element is `undefined`. 
If you prefer I can rewrite this line like this:
```
 map(({ editionLevels }) => editionLevels.length > 0 ? editionLevels[0] : undefined),
```
or 
```
 map(({ editionLevels }) => editionLevels[0] ?? undefined),
```